src/sage/schemes/curves/affine_curve.py

```diff
@@ -1762,7 +1808,7 @@ def fundamental_group(self, simplified=True, puiseux=False):
 
         - ``simplified`` -- (default: ``True``) boolean to simplify the presentation.
 
-        - ``puiseux`` -- (default: ``False``) boolean to decide if the
+        - ``puiseux`` -- (default: ``True``) boolean to decide if the
           presentation is constructed in the classical way or using Puiseux
           shortcut. If ``True``, ``simplified`` is set to ``False``.
```

Comment:
This makes the default arguments inconsistent. This change also needs some justification.

Reply:
You are right. Since `puiseux` was added later, I preferred to keep it as optional, but it is usually faster, so finally it is the default option. On the other side, the unsimplified presentation gives more information since the homotopy type of the complement is also encoded. The theoretical simplification moves can change the information about the homotopy type, but actually, the only ones that can do this are the elimination of trivial relations. If this happen the homotopy type can be retreived. One solution is to keep the default consistent and to add something as the Euler characteristic; with these data the homotopy type can be obtained. What do you think?

Reply:
Okay, that is a good reason for the change. The main thing is then changing `simplified` to have a default of `True` to be consistent with `puiseux` being `True` setting `simplified` to `True`.

Reply:
Sorry, I forgot to eliminate it,  now both settings are independent.